clients/cloud/python/README.md

```diff
@@ -17,6 +17,24 @@ sasl.username=<api-key-id>
 sasl.password=<secret-access-key>
 ```
 
+* Properly configured trust store on client system
+
+In order for the producer to verify the certificate of the brokers make sure the trust store is set up correctly on your system. 
```

Comment:
Add something like: "Depending on your operating system or Linux distro you may need to take extra steps to set up the SSL CA root certificates.. "

Reply:
Fixed.